### PR TITLE
Read only tabs when creating an item from a template

### PR DESCRIPTION
**Pull request by yllen**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #2322

*Please update this template with something that matches your PR*

### CODE REVIEW

**Comment by yllen** [comment]
Need to check all tab for all template

**Comment by yllen** [comment]
Computer template done

**Comment by yllen** [comment]
I think i checked every object creation with a template

File: inc/contract_item.class.php [hunk 1]
```diff
@@ -524,11 +523,12 @@ static function showForItem(CommonDBTM $item, $withtemplate='') {
     *
     * @since version 0.84
     *
-    * @param $contract   Contract object
+    * @param           $contract       Contract object
```
Comment (trasher) [inline inc/contract_item.class.php]: Missing type

File: inc/ticket.class.php [hunk 2]
```diff
@@ -5242,11 +5242,12 @@ static function showCentralNewList() {
     *
     * Will also display tickets of linked items
     *
-    * @param $item CommonDBTM object
+    * @param           $item          CommonDBTM object
```
Comment (trasher) [inline inc/ticket.class.php]: Missing type

**Pull request merged**